<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Runview</class>
 <widget class="QWidget" name="Runview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>531</width>
     <height>742</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="horizontalSpacing">
     <number>10</number>
    </property>
    <property name="verticalSpacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="label0">
      <property name="text">
       <string>本地存图:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label1">
      <property name="text">
       <string>相机AB延时(ms)：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label2">
      <property name="text">
       <string>相机CD延时(ms)：</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label3">
      <property name="text">
       <string> 屏蔽壳长度(ms)：</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label4">
      <property name="text">
       <string> 铁屑过滤宽度(ms)：</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>确认设置</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEdit_4"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="statusLabel">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
